### ISSUE TITLE
Livarno Lux E27-  CCT Bulb

### ISSUE THREAD

**Issue by Zazou89** [dresden-elektronik/deconz-rest-plugin#8062]
### Is there already an existing issue for this?

- [X] I have searched the existing issues and there is none for my device

### Product name

Livarno Lux E27 9W CCT Bulb

### Manufacturer

LIDL Livarno Lux

### Model identifier

HG06492C

### Device type to add

Light

### Node info

![image](https://github.com/user-attachments/assets/a9f18a02-c95f-4b00-9907-bfcbb801c98a)

### Endpoints and clusters

![image](https://github.com/user-attachments/assets/3f5ccad6-bdc0-4960-ae96-a18f306394d0)

### Basic

![image](https://github.com/user-attachments/assets/00555427-f147-4317-a908-fde694599357)

### On/Off
![image](https://github.com/user-attachments/assets/8ce5b410-e2a3-48ae-b14f-de29598b7a92)
![image](https://github.com/user-attachments/assets/2c84370e-40fe-42ba-a7ce-b16a62e4a068)

### Level Control
![image](https://github.com/user-attachments/assets/174a22ce-aa1f-4f3f-bf56-401db8e3e967)

### Color Control
![image](https://github.com/user-attachments/assets/33334dde-9431-4e60-ab04-88c9e13ae43a)
![image](https://github.com/user-attachments/assets/f7e025f2-265e-4770-b452-daad55c2b7cf)
![image](https://github.com/user-attachments/assets/4bedfb63-b041-4d31-92b7-c996d696dd05)
![image](https://github.com/user-attachments/assets/1c75d327-7d90-477c-8609-d8fce479931d)
![image](https://github.com/user-attachments/assets/b7871dda-7ff4-40e2-8b67-1121794c8cca)
![image](https://github.com/user-attachments/assets/1b03ce9a-0de5-4ffb-a5e9-6c1755d5200f)

**Comment by Smanar** [comment]
Hello, but the support is not native for this kind of device ?

**Comment by Zazou89** [comment]
> Hello, but the support is not native for this kind of device ?

I assumed it wasn't 100% integrated because I get warning messages for these devices in Homenridge (legacy code warning / XY gamut). 

Similarly, I don't know if this has anything to do with it, but in the DeconZ GUI it doesn't say "DDF" on the top right for these devices.

**Comment by Smanar** [comment]
Nope, you are right, this device don't have DDF, but most of bulbs are working natively.
